### ISSUE TITLE
integrate with NUnit Test Adapter for Visual Studio 

### ISSUE THREAD

**Issue by wangvnn** [fschwiet/DreamNJasmine#20]
It would be nice if NJasmine works with NUnitest Test Adapter.
http://nunit.org/index.php?p=vsTestAdapter&r=2.6

I use NJasmine because I cannot find anything out there that can do simple but effective BDD.
The real blocker for NJasmine maybe the test runner which may be fixed with the Adapter so that people can use VS text explorer and runner.

Thank you for wonderful work. Let me know if I can help to work on the adapter. I am learning VS extension now.

**Comment by fschwiet** [comment]
I did have NJasmine working in VS2012's test runner (see branch https://github.com/fschwiet/DreamNJasmine/tree/vs2012).  I forget how "finished"/"stable" it was.  I abandoned the effort because NJasmine tests tend to have long names, and the Visual Studio UI did not handle them well at the time.

I don't know if the VsTestAdapter would be an easier approach.  Feel free to work on it, though to be honest I probably won't be participating myself.  Its been so long I don't even think I can offer much guidance.  The psake file does have some VS2012 specific tests that can be run, that might be the place to start (see https://github.com/fschwiet/DreamNJasmine/commit/09c5d3c5f0adb4ff18833da925d485de797bd2b2 where they were disabled).
